docs/source/optim.rst

```diff
@@ -146,6 +146,38 @@ allows dynamic learning rate reducing based on some validation measurements.
 Learning rate scheduling should be applied after optimizer's update; e.g., you
 should write your code this way:
 
+Example::
+
+    scheduler = ExponentialLR(optimizer, gamma=0.9)
+    for epoch in range(20):
+        for input, target in dataset:
+            optimizer.zero_grad()
+            output = model(input)
+            loss = loss_fn(output, target)
+            loss.backward()
+            optimizer.step()
+            scheduler.step()
+
+One interesting feature of schedulers are that they can be called back-to-back and to
+get chained. This will lead to by schedulers are getting applied simultaneously.
```

Comment:
nit: I'd suggest removing "interesting feature" and highlighting that "most schedulers" support that feature (but not all). "simultaneously" also seems unclear to me: does it mean one override the other? I suggest:

> Most learning rate schedulers can be called back-to-back (also referred to as chaining schedulers). The result is that each scheduler is applied one after the other on the learning rate obtained by the one preceding it.